client/src/pages/record.tsx

```diff
@@ -167,6 +214,65 @@ function RecordPage(props: {
   const warnEmptyTranscript =
     curAnswer?.attentionNeeded === AnswerAttentionNeeded.NEEDS_TRANSCRIPT;
 
+  const [transcriptText, setTranscriptText] = useState<string>("");
+  const [editorState, setEditorState] = useState<EditorState>(
+    EditorState.createWithContent(ContentState.createFromText(""))
+  );
+  const markdownConfig = {
+    blockTypesMapping: {
+      /* mappings */
+    },
+    emptyLineBeforeBlock: true,
+  };
+
+  // Used to get the initial plain text
+  function getTranscriptText(text: string) {
```

Comment:
this function does much more than just getting the transcript text, also, you don't need a getter for the transcript text because you "get" it just by using the "transcriptText" variable

Reply:
It actually is supposed to say updateTranscriptText since it's used to change the state of this useEffect:

```
useEffect(() => {
    if (!curAnswer) {
      return;
    }
    updateTranscriptText(curAnswer.answer.transcript);
  }, [curAnswer?.answer]);
```